build.gradle

```
@@ -1,6 +1,6 @@
 plugins {
     id 'groovy'
-    id 'fabric-loom' version '1.6-SNAPSHOT'
+    id 'fabric-loom' version '1.8-SNAPSHOT'
```

I can't seem to get `genSources` to work with `fabric-loom` @ 1.8, so I'd recommend downgrading to 1.7 unless it breaks something else (doesn't appear to?).

[Related discord messages](https://canary.discord.com/channels/507304429255393322/842691768175951942/1299719177325641748)

```suggestion
    id 'fabric-loom' version '1.7-SNAPSHOT'
```